server/src/main/java/org/elasticsearch/cluster/metadata/MetadataCreateIndexService.java

```diff
@@ -306,12 +303,20 @@ public void onFailure(String source, Exception e) {
             });
     }
 
+    private void applyCreateIndexRequestInternal(CreateIndexClusterStateUpdateRequest createIndexClusterStateRequest){
```

Comment:
Calling this "apply" I think is a little misleading, I think `normalizeRequestSettings` would be a better name for this method